src/Utils/force_machine_learning_module.f95

```diff
@@ -640,7 +640,7 @@ subroutine write_distance_file_real(d, data_dir)
 !   OPEN(2, status='replace',file=trim(data_dir)//'Dist.dat',  form='UNFORMATTED')
 !   !write(2)  d
 !   close(2)
-    call fwrite_array_d(1, 0.0_dp, trim(data_dir)//'Dist.dat')
+    call fwrite_array_d(1, (/0.0_dp/), trim(data_dir)//'Dist.dat')
```

Comment:
This change is probably find.  `fwrite_array_d` expects a pointer to the beginning of the data, and since fortran passes everything by reference it should create a temporary to hold the 1-element array with 0 and pass that.